atlasdb-conjure/src/main/java/com/palantir/atlasdb/http/v2/DialogueShimFactory.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.http.v2;
+
+import com.palantir.conjure.java.client.jaxrs.JaxRsClient;
+import com.palantir.conjure.java.dialogue.serde.DefaultConjureRuntime;
+import com.palantir.dialogue.Channel;
+
+public final class DialogueShimFactory {
+    private DialogueShimFactory() {
+        // Nö
+    }
+
+    public static <T> T create(Class<T> type, Channel channel) {
```

Comment:
only used in one place... inline?

Reply:
I tried to write it that way at first... but most of the HTTP libs aren't visible in the `atlasdb-config` project (which seems kind of reasonable - that's the intention behind the `atlasdb-conjure` project) so this class was needed